app/services/users.js

```diff
@@ -0,0 +1,23 @@
+const User = require('../models/').User,
```

Comment:
use destructuring

Reply:
I read that:
const User = require('../Models/User').User;
const {User} = require('../Models/User');
Are the same thing.

I tried to understand why but I don't get it. I will ask you by Slack.

Reply:
yes, its the same. Its just cleaner in the long term to use destructuring. It does the same thing though